orchagent/bufferorch.cpp

```diff
@@ -579,7 +579,19 @@ task_process_status BufferOrch::processQueue(Consumer &consumer)
     }
     else
```

Comment:
Curious!
When will this port get into m_ready_list ?
May be when the port is marked up ?